pkg/storaged/storage.scss

```diff
@@ -31,12 +31,11 @@
 #main-storage.pf-c-page,
 #storage-detail .pf-c-page__main-section,
 #storage-detail .pf-c-page__main-nav {
-    padding: var(--pf-global--gutter);
     overflow: auto;
 }
 
 #main-storage > .pf-c-page__main {
-    padding-bottom: var(--pf-global--spacer--md);
+    padding: var(--pf-global--gutter);
```

Comment:
This should be fixed by adding the PageSection component around the grid. In pkg/storaged/overview.jsx This will also fix the padding to be the same like all other pages, now it's smaller.

Reply:
<details>
  <summary>Click to expand!</summary>
  
``` diff
$ git diff
diff --git pkg/storaged/overview.jsx pkg/storaged/overview.jsx
index 65a8e91a3..70ef05ccb 100644
--- pkg/storaged/overview.jsx
+++ pkg/storaged/overview.jsx
@@ -19,7 +19,7 @@
 
 import React from "react";
 
-import { Page, Grid, GridItem, Card, CardBody, Gallery } from "@patternfly/react-core";
+import { Page, PageSection, Grid, GridItem, Card, CardBody, Gallery } from "@patternfly/react-core";
 
 import { StoragePlots } from "./plot.jsx";
 
@@ -37,30 +37,32 @@ import { StorageLogsPanel } from "./logs-panel.jsx";
 export const Overview = ({ client, plot_state }) => {
     return (
         <Page id="main-storage">
-            <Grid hasGutter>
-                <GridItem md={8} lg={9}>
-                    <Gallery hasGutter>
-                        <Card>
-                            <CardBody>
-                                <StoragePlots plot_state={plot_state} />
-                            </CardBody>
-                        </Card>
-                        <FilesystemsPanel client={client} />
-                        <LockedCryptoPanel client={client} />
-                        <NFSPanel client={client} />
-                        <JobsPanel client={client} />
-                        <StorageLogsPanel />
-                    </Gallery>
-                </GridItem>
-                <GridItem md={4} lg={3} className="storage-sidebar">
-                    <Gallery hasGutter>
-                        <ThingsPanel client={client} />
-                        <DrivesPanel client={client} />
-                        <IscsiPanel client={client} />
-                        <OthersPanel client={client} />
-                    </Gallery>
-                </GridItem>
-            </Grid>
+            <PageSection>
+                <Grid hasGutter>
+                    <GridItem md={8} lg={9}>
+                        <Gallery hasGutter>
+                            <Card>
+                                <CardBody>
+                                    <StoragePlots plot_state={plot_state} />
+                                </CardBody>
+                            </Card>
+                            <FilesystemsPanel client={client} />
+                            <LockedCryptoPanel client={client} />
+                            <NFSPanel client={client} />
+                            <JobsPanel client={client} />
+                            <StorageLogsPanel />
+                        </Gallery>
+                    </GridItem>
+                    <GridItem md={4} lg={3} className="storage-sidebar">
+                        <Gallery hasGutter>
+                            <ThingsPanel client={client} />
+                            <DrivesPanel client={client} />
+                            <IscsiPanel client={client} />
+                            <OthersPanel client={client} />
+                        </Gallery>
+                    </GridItem>
+                </Grid>
+            </PageSection>
         </Page>
     );
 };
diff --git pkg/storaged/storage.scss pkg/storaged/storage.scss
index b80be8273..36225c06e 100644
--- pkg/storaged/storage.scss
+++ pkg/storaged/storage.scss
@@ -34,10 +34,6 @@
     overflow: auto;
 }
 
-#main-storage > .pf-c-page__main {
-    padding: var(--pf-global--gutter);
-}
-
 #storage .create-storage {
     margin-bottom: 10px;
 }
```
</details>

Reply:
Ah cool.